lib/sisimai/bite/email/verizon.rb

```diff
@@ -0,0 +1,265 @@
+module Sisimai::Bite::Email
+  module Verizon
+    # Sisimai::Bite::Email::Verizon parses a bounce email which created by
+    # Verizon Wireless.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Verizon.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->